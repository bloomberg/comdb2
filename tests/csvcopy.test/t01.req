delete from t1 where 1

exec procedure sys.cmd.load('DBDIR/agg1.csv', 't1')

select * from t1 order by name

exec procedure sys.cmd.load('DBDIR/agg1.csv', 't1', ':')

select * from t1 order by name

delete from t1 where 1

exec procedure sys.cmd.load('DBDIR/agg1.csv', 't1',':', 'name:value:time')

select * from t1 order by name
