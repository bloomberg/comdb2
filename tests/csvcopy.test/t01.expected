(rows deleted=5)
[exec procedure sys.cmd.load('DBDIR/agg1.csv', 't1')] failed with rc -3 [sys.load(db,csv,tbl,seperator,h...]:2: table t1 has no column named name:value:time in stmt: INSERT INTO "t1" ("name:value:time") VALUES (?)
(name='ab21cd', value='ef23sdss', time='k223')
(name='absdcd', value='ee1fsdss', time='kss')
(name='dfgjhk', value='efsdss', time='k23')
(name='dh42bcd', value='efsfsdss', time='k42')
(rows deleted=4)
(name='ab21cd', value='ef23sdss', time='k223')
(name='absdcd', value='ee1fsdss', time='kss')
(name='dfgjhk', value='efsdss', time='k23')
(name='dh42bcd', value='efsfsdss', time='k42')
(name='name', value='value', time='time')
